all/pom.xml

```diff
@@ -284,6 +284,11 @@
             <artifactId>seata-compressor-zip</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
pls add `lz4-java` dependency in all.pom

Reply:
OK👌
Please review.